src/modules/objectToLS.js

```diff
@@ -0,0 +1,13 @@
+import { store, taskArray } from './store.js';
+
+const addObjToLocalStorage = (value) => {
+  const object = {
```

Comment:
I think it would be better to change the variable name `object` to `task` or `todo` to provide more precise context.